src/pyramid/view.py

```diff
@@ -23,6 +23,7 @@
 from pyramid.util import hide_attrs, reraise as reraise_
 
 _marker = object()
+LIFT = object()
 
```

Comment:
The constant needs to be exposed in the api docs at the very least, plus a changelog entry explaining the feature.